apps/demo/package.json

```diff
@@ -33,12 +33,13 @@
     "react-router-dom": "6.3.0"
   },
   "devDependencies": {
+    "@types/node": "16.11.7",
```

Comment:
Now that the `.eslintrc.js` files are linted, we need to install Node's types package so galex knows to enable ESLint's `node` environment (which includes global variables such as `module`, `require`, etc.)